supervised/pomodoro/src/argus_pomodoro.py

```diff
@@ -0,0 +1,209 @@
+API_TEMPLATE = f"api/v{API_VERSION}/"
+
+
+# argus API
+
+@contextmanager
+def translate_api_error(context=None):
+    error_msg = ""
+    try:
+        yield
+    except httpx.ConnectError as e:
+        error_msg = f"Connection error: {e}"
+    except httpx.UnsupportedProtocol as e:
+        error_msg = f"Unsupprted protocol: {e}"
+    except exceptions.ErrorWithResponse as e:
+        response = e.response
+        error_msg = f"{response.status_code} {response.client_response.reason_phrase} ({response.url})"
+        if isinstance(response.body, dict):
+            # HTTP status codes that have no content
+            detail = response.body.get("detail", "")
+            error_msg += f": {detail}"
+    except exceptions.ClientConnectionError as e:
+        error_msg = f"Failed to connect: {e}"
+    except exceptions.ActionNotFound as e:
+        error_msg = f"Unknown API call: {e}"
+    except exceptions.ActionURLMatchError as e:
+        error_msg = f"Unknown API endpoint: {e}"
+
+    if not error_msg:
+        return
+
+    sys.stderr.write(error_msg + "\n")
+    if context:
+        sys.stderr.write(f"More context: {context}\n")
+    sys.stderr.flush()
+
+    sys.exit(1)
+
+
+def check_connection(client, config):
+    with translate_api_error("Failed to connect on check"):
+        next(client.get_my_incidents())
+    if config.debug:
+        print("Checking: host and token ok")
+    sys.exit(0)
+
+
+def find_previous_break_incident(client):
+    incidents = list(client.get_my_incidents(open=True))
+    assert 0 <= len(incidents) < 2, "Too many open pomodoro incidents!"
+    if not incidents:
+        return None
+    if len(incidents) == 1:
+        return incidents[0]
+    # TBD: there's been an error, close all irrelevant incidents
+
+
+def start_break_incident(client, config):
+    tags = {
+        "break_duration": config.break_duration,
+        "time_between_breaks": config.work_duration,
+    }
+    incident = Incident(
+        description="Break time!",
+        start_time=datetime.now().astimezone(),
+        tags=tags,
+    )
+    client.post_incident(incident)
+    return incident
+
+
+def stop_break_incident(incident, client):
+    client.resolve_incident(
+        incident=incident.pk,
+        description="Break over!",
+        timestamp=datetime.now().astimezone(),
+    )
+
+
+# configuration/argument parsing
+
+
+def make_argparser():
+    parser = argparse.ArgumentParser(
+        prog=__progname__,
+        description=__description__,
+    )
+
+    parser.add_argument("host", help="Argus API host url (with scheme)")
+    parser.add_argument("token", help="Token to authenticate with")
+    parser.add_argument(
+        "-b",
+        "--break-duration",
+        type=int,
+        help="How many minutes to take a break",
+        default=5,
+    )
+    parser.add_argument(
+        "-w",
+        "--work-duration",
+        type=int,
+        help="How many minutes between breaks",
+        default=15,
```

Comment:
```suggestion
        default=25,
```
Only because that's how long it is typically